src/_posts/databases/2000-01-01-backup-policies.md

```diff
@@ -0,0 +1,146 @@
+dashboard or the CLI, or rely on the automated and scheduled backup service,
+which runs every 24 hours.
+
+### Important Considerations
+
+The time required to complete a backup depends on the amount of data being
+backed up. This operation may impact the performance of the database and, by
+extension, the application. On Business plans, backups are performed on the
+secondary instance to limit the impact on the primary instance.
+
+However, several factors can delay, cancel, or impact the backup process. These
+include the database load, the volume of data stored, and the nature of the
+queries running during the backup. The service is optimized for standard use,
+but large datasets or specific operations may impede the backup's successful
+execution. In such cases, we encourage users to contact Scalingo support to
+explore alternative architectural solutions that could improve the scalability
+of their applications.
+
+#### Execution Conditions
+
+Some specific conditions may delay, cancel, or affect the consistency of the
+backup:
+
+- Backups are only performed if the feature is enabled, which is the default
+  setting. However, users can manually disable it if needed.
+- The database must be in a running state at the scheduled time of the backup.
+- No maintenance operations, whether scheduled or unscheduled, should be
+  performed during backup time. The backup could otherwise be delayed or
+  skipped entirely.
+- Application-level locks or long-running transactions can prevent backups from
+  executing, as these can block access to certain tables or resources needed
+  during the backup process.
+- The scheduled time is flexible: it might be delayed depending on the load on
+  our infrastructure.
+
+### Scheduled Backups
+
+Scalingo provides a scheduled (or periodic) backup service for databases in the
+Starter and Business plans. Available for all supported databases, scheduled
+backups are performed every 24 hours. The start time can be customized through
+the dashboard or via the API, enabling you to align backups with your workload
+and core business operations.
+
+#### Retention Policy for Scheduled Backups
+
+We retain a limited number of backups based on your database plan:
+
+- **Daily backups**: one backup is retained for each of the last 7 days, for a
+  total of 7 backups.
+- **Weekly backups**: the daily backup taken on Sunday is retained as the
+  weekly backup.
+- **Monthly backups**: the daily backup from the first Sunday of the month is
+  retained as the monthly backup.
+
+| Plan     | Daily Backups  | Weekly Backups  | Monthly Backups   |
+| -------- | -------------- | --------------- | ----------------- |
+| Sandbox  | None           | None            | None              |
+| Starter  | 7 rolling days | 4 rolling weeks | None              |
+| Business | 7 rolling days | 8 rolling weeks | 12 rolling months |
+
+Deleting a database doesn’t immediately remove its scheduled backups. They
```

Comment:
question: this is a weird character you used here for the quote. Do you remember how you copied it? :thinking: 

```suggestion
Deleting a database doesn't immediately remove its scheduled backups. They
```